### PR TITLE
UI enhancements

### PR DESCRIPTION
**Pull request by costowell**

### CODE REVIEW

File: src/components/ConfirmModal/ConfirmModal.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+interface ConfirmModalProps extends PropsWithChildren {
+    onConfirm: () => void,
+    headerText: string,
+    confirmText?: string,
+    cancelText?: string,
+    color?: string,
+    isOpen: boolean,
+    toggle: () => void,
+    quote?: Quote,
```
Comment (ethanf108) [inline src/components/ConfirmModal/ConfirmModal.tsx]: Is this used anywhere?

File: src/components/QuoteCard/QuoteCard.tsx [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faCaretDown, faCaretUp, faSquareCaretDown, faSquareCaretUp } from "@fortawesome/free-solid-svg-icons";
+import { faCaretDown, faCaretUp, faSquareCaretDown, faSquareCaretUp, faEllipsis } from "@fortawesome/free-solid-svg-icons";
 import { ReactNode, useEffect, useState } from "react";
 
 interface Props {
     quote: Quote,
     onVoteChange?: (type: Vote) => void,
-    children?: ReactNode,
+    children?: ReactNode[],
```
Comment (ethanf108) [inline src/components/QuoteCard/QuoteCard.tsx]: Why is this changed to an Array?

File: src/pages/Storage/Storage.tsx [hunk 3]
```diff
@@ -190,6 +228,42 @@ const Storage = (props: Props) => {
...
+                      confirmText="Report"
+                    >
+                      <QuoteCard quote={targetQuote!}/>
+                      <Input
+                          type="text"
+                          placeholder="Why do you want to report this Quote?"
+                          value={reportText}
+                          onChange={e => setReportText(e.target.value)}
+                      />
+                    </ConfirmModal>
```
Comment (ethanf108) [inline src/pages/Storage/Storage.tsx]: Is there any way we caan reduce the amount of code here? It looks like there's a lot of reused code.

**Review by ethanf108 — CHANGES_REQUESTED** [review]
Be sure to run `npm run lint` before committing. I added it to a git pre-commit hook, might be useful

File: src/components/QuoteCard/QuoteCard.tsx (hunk 2 above)
Comment (costowell) [reply src/components/QuoteCard/QuoteCard.tsx]: You right.

File: src/components/ConfirmModal/ConfirmModal.tsx (hunk 1 above)
Comment (costowell) [reply src/components/ConfirmModal/ConfirmModal.tsx]: You right.

File: src/pages/Storage/Storage.tsx (hunk 3 above)
Comment (costowell) [reply src/pages/Storage/Storage.tsx]: I suppose its possible, but I think it would just complicate the code base more.

**Review by ethanf108 — APPROVED** [review]
